HTAN.model.csv

```diff
@@ -1030,3 +1034,8 @@ Barretts Esophagus Goblet Cells Present,Presence or absennce of Barretts esophag
 Pancreatitis Onset Year,Date of onset of pancreatitis.,,,,FALSE,Follow Up,,,num
 HTAN Parent Channel Metadata ID,HTAN ID for a level 3 channels table.,,,,TRUE, Imaging Level 4,,,
 Single Nucleus Capture,Nuclei isolation method,"Plates, 10x, droplet",,,FALSE,scmC-seq Level 1,,,
+Associated mRNA Library Data File ID,Sample Level HTAN Data File ID for the associated level - HTAN ID of this file HTAN ID SOP (eg HTANx_yyy_zzz),,,,TRUE,Multiplexed CITE-seq Level 1,,,
+Single Cell Barcode Method Applied,The method by which cells are multiplex or labeled with cell surface markers or probes,"Cite-seq custom panel,Biolegend Total seq A Custom,Biolegend Total seq B Custom,Biolegend Total seq C Custom,Biolegend Total seq A Human,Universal cocktail V1.0,Biolegend Total seq B Human Universal cocktail V1.0,Biolegend Total seq C Human Universal cocktail V1.0,Cell,Hashtag- multiplex,Nuclear Hashtag- multiplex,Other",,,TRUE,Multiplexed CITE-seq Level 1,,,
+Feature Barcode Library Type,The library construction methods for the feature barcode library,"10x ADT – citeseq/hashing,10x TCR,10x BCR",,,TRUE,Multiplexed CITE-seq Level 1,,,
+Barcode Folder Synapse ID,Synapse ID of the folder containing the barcode lists,,,,TRUE,Multiplexed CITE-seq Level 1,,,

```

Comment:
suggest adding the validation rule `regex match syn\d+`

Reply:
OK, I've added this validation regex in.